src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -437,7 +437,7 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
 
   /** Grouped files in group order, and lex order within each group. */
   def groupedFiles(sources: List[File]): List[List[File]] = (
-    if (sources.tail.nonEmpty) {
+    if (sources.size > 1) {
```

Comment:
This is the wrong fix--`.size` is an O(n) operation on a list and this doesn't require that.  Instead, use `size_compare` or `sources.drop(1).nonEmpty` (both are cheap).

Reply:
or `sizeIs > 1`

Reply:
What crafty magic!